ext/cumo/include/cumo/narray.h

```diff
@@ -265,6 +267,23 @@ typedef struct {
     unsigned int element_stride;
 } cumo_narray_type_info_t;
 
+// from ruby/enumerator.c
+struct enumerator {
```

Comment:
Does not this conflict with Numo?

Reply:
I think it do not conflict as long as I have confirmed.
But, I added cumo_ prefix to avoid concerns of conflict with Numo.
Thanks.